server/certs/genCerts.sh

```diff
@@ -0,0 +1,23 @@
+# From https://github.com/joe-elliott/cert-exporter/blob/master/test/files/genCerts.sh
```

Comment:
Note this is under Apache 2 licence.
(also it's more future-proof to use a commit hash than `master` in the URL)

Reply:
Ah, right. I could send a follow up PR to change this.